runtime/stdlib/test.go

```diff
@@ -154,30 +161,51 @@ var blockchainBackendInterfaceType = func() *sema.InterfaceType {
 var matcherType = func() *sema.CompositeType {
 	typ, ok := testContractType.NestedTypes.Get(matcherTypeName)
 	if !ok {
-		panic(errors.NewUnexpectedError("cannot find type %s.%s", testContractTypeName, matcherTypeName))
+		panic(typeNotFoundError(testContractTypeName, matcherTypeName))
 	}
 
-	interfaceType, ok := typ.(*sema.CompositeType)
+	compositeType, ok := typ.(*sema.CompositeType)
 	if !ok {
-		panic(errors.NewUnexpectedError("invalid type for %s. expected interface", matcherTypeName))
+		panic(errors.NewUnexpectedError(
+			"invalid type for '%s'. expected struct type",
+			matcherTypeName,
+		))
 	}
 
-	return interfaceType
+	return compositeType
 }()
 
-var matcherTestFunctionType = func() *sema.FunctionType {
-	testFunc, ok := matcherType.Members.Get(matcherTestFunctionName)
+var matcherTestFunctionType = compositeFunctionType(matcherType, matcherTestFunctionName)
+
+func compositeFunctionType(parent *sema.CompositeType, funcName string) *sema.FunctionType {
+	testFunc, ok := parent.Members.Get(funcName)
 	if !ok {
-		panic(errors.NewUnexpectedError("cannot find type %s.%s", matcherTypeName, matcherTestFunctionName))
+		panic(typeNotFoundError(parent.Identifier, funcName))
```

Comment:
Should be a *member* not found error. Same below

Reply:
fixed in fbd4f3f490a2e254fbc04b985314b1383abdfccd